debezium-core/src/main/java/io/debezium/relational/ColumnEditor.java

```diff
@@ -39,6 +39,14 @@
     int jdbcType();
 
     /**
+     * Get the underlying type for this column where
+     * jdbcType() itself is the generic Types.ARRAY.
+     *
```

Comment:
How about having it return null -- or even raise an `IllegalStateException` or so -- when invoking it for a non-array type?